.github/workflows/test.yml

```diff
@@ -79,8 +98,12 @@ jobs:
     steps:
       - name: Get code
         uses: actions/checkout@v3
+        with:
+          repository: yash-zededa/eden
```

Comment:
Here as well you're referring to your fork

Reply:
Yes, I have repointed to my fork to test the workflow by manual trigger from eden.  It works now so we are good to point to lf-edge/eden.